README.md

```diff
@@ -34,18 +34,25 @@ You should see 1 passing test.
 
 ## User stories
 
+
 ```
 As a Maker
 So that I can see what people are doing
 I want to see all the messages (peeps)
 in a browser
 ```
 
+
+![alt text](story1diagram.png "See all messages diagram")
```

Comment:
I agree! It's always a good idea to diagram things out.

One thing I would note is that in these diagrams you've included both `app.rb` and `peeps.rb` under the Controller and under model you have the database itself.

That's not quite how the MVC model is meant to be understood. The database isn't really a part of the MVC model, the MVC model only covers the code you write:

- The Model part refers to the code you write that interacts with the database (peeps.rb) and that provides a way to represent the data once you've retrieved it from the database (one instance of the `Peep` class represents a single row of the DB)
- As your diagram already suggests, the Controller part refers to the code you write that sets up routes and is the link between your View and your Model. It's just that in your codebase, this means that the Controller is really only the code in the `app.rb` file (in more complex codebases it could cover more files, but there's no need to worry about that at this point in the course!)